Extended series metadata do not work with MariaDB

**Describe the bug**
A clear and concise description of what you see:
*When i add an extend metadata catalog in serie i can display my metadata II in the interface, but soon as i set some data inside, it's crashing*

**To Reproduce**
Steps to reproduce the behavior:
1. Go to 'opencast and add an extended metadata catalog'
2. Add a test serie without extend metadata, everything is fine
3. Add a data in your metadata II
4. See error in your log (or in my log in this case)

```
2022-06-29T07:17:49,618 | ERROR | (ConfigurableSeriesDCCatalogUIAdapter:104) - Error while loading DublinCore catalog of series '54cc3d1d-4445-491b-9c7d-59410806f4cf': {}
org.opencastproject.series.api.SeriesException: org.opencastproject.series.impl.SeriesServiceDatabaseException: java.lang.ClassCastException: class org.mariadb.jdbc.MariaDbBlob cannot be cast to class [B (org.mariadb.jdbc.MariaDbBlob is in unnamed module of loader org.apache.felix.framework.BundleWiringImpl$BundleClassLoader @40f2cab4; [B is in module java.base of loader 'bootstrap')
        at org.opencastproject.series.impl.SeriesServiceImpl.getSeriesElementData(SeriesServiceImpl.java:458) ~[?:?]
        at org.opencastproject.index.service.catalog.adapter.series.ConfigurableSeriesDCCatalogUIAdapter.loadDublinCoreCatalog(ConfigurableSeriesDCCatalogUIAdapter.java:87) [!/:?]
        at org.opencastproject.index.service.catalog.adapter.series.ConfigurableSeriesDCCatalogUIAdapter.getFields(ConfigurableSeriesDCCatalogUIAdapter.java:62) [!/:?]
        at org.opencastproject.adminui.endpoint.SeriesEndpoint.getSeriesMetadata(SeriesEndpoint.java:329) [!/:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
        at org.apache.cxf.service.invoker.AbstractInvoker.performInvocation(AbstractInvoker.java:179) [!/:3.4.6]
        at org.apache.cxf.service.invoker.AbstractInvoker.invoke(AbstractInvoker.java:96) [!/:3.4.6]
        at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:201) [!/:3.4.6]
        at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:104) [!/:3.4.6]
        at org.apache.cxf.interceptor.ServiceInvokerInterceptor$1.run(ServiceInvokerInterceptor.java:59) [!/:3.4.6]
        at org.apache.cxf.interceptor.ServiceInvokerInterceptor.handleMessage(ServiceInvokerInterceptor.java:96) [!/:3.4.6]
        at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:308) [!/:3.4.6]
        at org.apache.cxf.transport.ChainInitiationObserver.onMessage(ChainInitiationObserver.java:121) [!/:3.4.6]
        at org.apache.cxf.transport.http.AbstractHTTPDestination.invoke(AbstractHTTPDestination.java:265) [!/:3.4.6]
        at org.apache.cxf.transport.servlet.ServletController.invokeDestination(ServletController.java:234) [!/:3.4.6]
        at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:208) [!/:3.4.6]
        at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:160) [!/:3.4.6]
        at org.apache.cxf.transport.servlet.CXFNonSpringServlet.invoke(CXFNonSpringServlet.java:225) [!/:3.4.6]
        at org.apache.cxf.transport.servlet.AbstractHTTPServlet.handleRequest(AbstractHTTPServlet.java:304) [!/:3.4.6]
        at org.opencastproject.kernel.rest.RestPublisher$RestServlet.handleRequest(RestPublisher.java:615) [!/:?]
        at org.apache.cxf.transport.servlet.AbstractHTTPServlet.doGet(AbstractHTTPServlet.java:222) [!/:3.4.6]
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:687) [!/:3.1.0]
        at org.apache.cxf.transport.servlet.AbstractHTTPServlet.service(AbstractHTTPServlet.java:279) [!/:3.4.6]
        at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:799) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.servlet.ServletHandler$ChainEnd.doFilter(ServletHandler.java:1626) [!/:9.4.43.v20210629]
        at org.opencastproject.kernel.rest.JsonpFilter.doFilter(JsonpFilter.java:125) [!/:?]
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601) [!/:9.4.43.v20210629]
        at org.opencastproject.kernel.filter.proxy.TransparentProxyFilter.doFilter(TransparentProxyFilter.java:75) [!/:?]
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601) [!/:9.4.43.v20210629]
        at org.opencastproject.security.urlsigning.filter.UrlSigningFilter.doFilter(UrlSigningFilter.java:100) [!/:?]
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601) [!/:9.4.43.v20210629]
        at org.opencastproject.kernel.security.RemoteUserAndOrganizationFilter.doFilter(RemoteUserAndOrganizationFilter.java:260) [!/:?]
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601) [!/:9.4.43.v20210629]
        at org.opencastproject.kernel.rest.CurrentJobFilter.doFilter(CurrentJobFilter.java:101) [!/:?]
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601) [!/:9.4.43.v20210629]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:330) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.invoke(FilterSecurityInterceptor.java:118) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.doFilter(FilterSecurityInterceptor.java:84) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [!/:3.1.7.RELEASE]
        at org.opencastproject.kernel.security.AsyncTimeoutRedirectFilter.doFilter(AsyncTimeoutRedirectFilter.java:60) [!/:?]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:113) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:103) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:113) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.authentication.rememberme.RememberMeAuthenticationFilter.doFilter(RememberMeAuthenticationFilter.java:146) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:54) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:45) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [!/:3.1.7.RELEASE]
        at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:98) [!/:3.1.4.RELEASE]
        at org.springframework.security.web.authentication.www.BasicAuthenticationFilter.doFilter(BasicAuthenticationFilter.java:150) [!/:3.1.7.RELEASE]
        at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:102) [!/:3.1.4.RELEASE]
        at org.springframework.security.web.authentication.www.DigestAuthenticationFilter.doFilter(DigestAuthenticationFilter.java:115) [!/:3.1.7.RELEASE]
        at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:102) [!/:3.1.4.RELEASE]
        at org.springframework.web.filter.CompositeFilter.doFilter(CompositeFilter.java:82) [!/:3.1.4.RELEASE]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.authentication.ui.DefaultLoginPageGeneratingFilter.doFilter(DefaultLoginPageGeneratingFilter.java:91) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:183) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:105) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:87) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:192) [!/:3.1.7.RELEASE]
        at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:160) [!/:3.1.7.RELEASE]
        at org.opencastproject.kernel.security.SecurityFilter.doFilter(SecurityFilter.java:138) [!/:?]
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601) [!/:9.4.43.v20210629]
        at org.opencastproject.kernel.security.OrganizationFilter.doFilter(OrganizationFilter.java:149) [!/:?]
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601) [!/:9.4.43.v20210629]
        at org.opencastproject.kernel.filter.https.HttpsFilter.doFilter(HttpsFilter.java:85) [!/:?]
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601) [!/:9.4.43.v20210629]
        at org.opencastproject.kernel.rest.CleanSessionsFilter.doFilter(CleanSessionsFilter.java:101) [!/:?]
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:201) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.websocket.server.WebSocketUpgradeFilter.doFilter(WebSocketUpgradeFilter.java:228) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:548) [!/:9.4.43.v20210629]
        at org.ops4j.pax.web.service.jetty.internal.HttpServiceServletHandler.doHandle(HttpServiceServletHandler.java:71) [!/:?]
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:602) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:235) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1624) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1434) [!/:9.4.43.v20210629]
        at org.ops4j.pax.web.service.jetty.internal.HttpServiceContext.doHandle(HttpServiceContext.java:298) [!/:?]
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:501) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1594) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1349) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141) [!/:9.4.43.v20210629]
        at org.ops4j.pax.web.service.jetty.internal.JettyServerHandlerCollection.handle(JettyServerHandlerCollection.java:82) [!/:?]
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.server.Server.handle(Server.java:516) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:388) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:633) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:380) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:277) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883) [!/:9.4.43.v20210629]
        at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034) [!/:9.4.43.v20210629]
        at java.lang.Thread.run(Thread.java:829) [?:?]
Caused by: org.opencastproject.series.impl.SeriesServiceDatabaseException: java.lang.ClassCastException: class org.mariadb.jdbc.MariaDbBlob cannot be cast to class [B (org.mariadb.jdbc.MariaDbBlob is in unnamed module of loader org.apache.felix.framework.BundleWiringImpl$BundleClassLoader @40f2cab4; [B is in module java.base of loader 'bootstrap')
        at org.opencastproject.series.impl.persistence.SeriesServiceDatabaseImpl.getSeriesElement(SeriesServiceDatabaseImpl.java:806) ~[?:?]
        at org.opencastproject.series.impl.SeriesServiceImpl.getSeriesElementData(SeriesServiceImpl.java:456) ~[?:?]
        ... 121 more
Caused by: java.lang.ClassCastException: class org.mariadb.jdbc.MariaDbBlob cannot be cast to class [B (org.mariadb.jdbc.MariaDbBlob is in unnamed module of loader org.apache.felix.framework.BundleWiringImpl$BundleClassLoader @40f2cab4; [B is in module java.base of loader 'bootstrap')
        at org.opencastproject.series.impl.persistence.SeriesServiceDatabaseImpl.getSeriesElement(SeriesServiceDatabaseImpl.java:800) ~[?:?]
        at org.opencastproject.series.impl.SeriesServiceImpl.getSeriesElementData(SeriesServiceImpl.java:456) ~[?:?]
        ... 121 more
```

**Environment Information**
 - Server OS: [Ubuntu 22.04 LTS (GNU/Linux 5.15.0-1014-aws x86_64)]
 - Browser [all]
 - Opencast version with minor release [Opencast 12.0.0 - 085d2c6 - built on 2022-06-28]

--- a/modules/db/pom.xml
+++ b/modules/db/pom.xml
@@ -59,7 +59,7 @@
     <dependency>
       <groupId>org.mariadb.jdbc</groupId>
       <artifactId>mariadb-java-client</artifactId>
-      <version>3.0.4</version>
+      <version>3.0.5</version>
     </dependency>
     <dependency>
       <groupId>org.postgresql</groupId>

